modules/javafx.base/src/main/java/javafx/util/FXPermission.java

```diff
@@ -43,7 +43,12 @@
  * @see java.security.PermissionCollection
  *
  * @since 9
+ *
+ * @deprecated This class was only useful in connection with the
+ *       Security Manager, which is no longer supported.
```

Comment:
should it be SecurityManager (as in class name)?

Reply:
No, this refers to "the Security Manager" as a feature, which is more than the `SecurityManager` class. See https://openjdk.org/jeps/486.